MULTIPLATFORM.md

```diff
@@ -0,0 +1,325 @@
+
+    actual class Platform actual constructor() {
+        actual val platform: String = "Android"
+    }
+
+We build this project into a Java-library which can be used by Android Studio project as a dependency. So the content
+of `android/build.gradle` will be the following:
+
+    apply plugin: 'java-library'
+    apply plugin: 'kotlin-platform-jvm'
+
+    // Specify a group and a version of the library to access it in Android Studio.
+    // By default the project directory name is used as an artefact name thus the full dependency
+    // description will be 'org.greeting:android:1.0'
+    group = 'org.greeting'
+    version = 1.0
+
+    dependencies {
+        // Specify Kotlin/JVM stdlib dependency.
+        implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+
+        // Specify dependency on a common project for Kotlin multiplatform build.
+        expectedBy project(':greeting:common')
+    }
+
+#### 2.3 iOS subproject
+
+As well as `android`, this project contains platform-dependent implementations of `expect`-declarations:
+
+    // greeting/ios/src/main/kotlin/ios.kt
+    package org.greeting
+
+    actual class Platform actual constructor() {
+        actual val platform: String = "iOS"
+    }
+
+We build this project into an Objective-C framework using Kotlin/Native compiler. To do this, declare a framework in
+`ios/build.gradle` and add an `expectedBy` dependency in the same manner as in the Android project:
+
+    apply plugin: 'konan'
+
+    // Specify targets to build the framework: iOS and iOS simulator
+    konan.targets = ['iphone', 'iphone_sim']
+
+    konanArtifacts {
+        // Declare building into a framework.
+        framework('iosGreeting') {
+            // The multiplatform support is disabled by default.
+            enableMultiplatform true
+        }
+    }
+
+    dependencies {
+        // Specify dependency on a common project for Kotlin multiplatform build
+        expectedBy project(':greeting:common')
+    }
+
+### 3. Android application
+
+Now we can create an Android application which will use the library we implemented on the previous step. Open Android
+Studio and create a new project in the `androidApp` directory. Android Studio generates all necessary files and
+directories so we only need to add a dependency on our library. There are 2 actions we need to do:
+
+1. Add dependency on the library. To do this just open `androidApp/app/build.gradle` and add the following snippet in
+the `dependencies` script block:
+
+    ```
+    implementation 'org.greeting:android:1.0'
+    ```
+2. Include `greeting` build in the Android Studio project as a part of
+[composite build](https://docs.gradle.org/current/userguide/composite_builds.html). To do this, add the
+following line in `androidApp/settings.gradle`:
+
+    ```
+    includeBuild '../greeting'
+    ```
+    Now dependencies of the application can be resolved in artefacts built by `greeting`. You also may publish the
```

Comment:
Isn't it called [`artifacts`](http://grammarist.com/spelling/artefact-artifact/)? 🤔

Reply:
Looks like both version are ok: https://english.stackexchange.com/questions/37903/difference-between-artifact-and-artefact.
But you're right from another point of view: it's better to have one spelling over all the documentation. I've replaced `artefact` -> `artifact`.